src/dirty_lemmas.k.md

```diff
@@ -1,2 +1,4 @@
 ```k
+rule X modInt pow32 => X
+    requires #rangeUInt(32, X)
```

Comment:
i use this generalized rule instead:
```
rule X modInt Y => X
  requires X <Int Y
```

Reply:
This should probably go to `rules.k.tmpl`, right?